### PR TITLE
COG-1258 Unlock locked users

### PR DESCRIPTION
**Pull request by sunilb0575**
#### Which User story does this relate to (link)?
- https://osi-cwds.atlassian.net/browse/COG-1164
- https://osi-cwds.atlassian.net/browse/COG-1258  (FE - Sub-task)

#### Brief feature description
- As an admin unlock locked user, when have edit privileges.

### CODE REVIEW

File: app/javascript/selectors/detailSelector.test.js [hunk 1]
```diff
@@ -341,32 +346,131 @@ describe('selectors', () => {
...
     })
 
-    it('return false if editable is not available ', () => {
+    it('returns false when editable & locked are not available ', () => {
       const state = getState({ fetchDetails: {} })
       expect(isUserEditable(state)).toEqual(false)
     })
   })
 
+  describe('#isAdminWithEditPrivileges', () => {
```
Comment (RamuKammagani-osi) [inline app/javascript/selectors/detailSelector.test.js]: Can you add more specs here for null and undefined cases, like if ```isDetailsEditable: null & undefined```, we found multiple times that page has broken as we dint cover those two cases in selectors, so just to make sure!

File: app/javascript/selectors/detailSelector.test.js [hunk 2]
```diff
@@ -341,32 +346,131 @@ describe('selectors', () => {
...
+    })
+
+    it('returns undefined if editable is not available', () => {
+      const state = getState({ fetchDetails: {} })
+      expect(isAdminWithEditPrivileges(state)).toEqual(undefined)
+    })
+  })
+
+  describe('#isUserLocked', () => {
+    it('returns true when locked is true', () => {
```
Comment (RamuKammagani-osi) [inline app/javascript/selectors/detailSelector.test.js]: same here please

File: app/javascript/selectors/detailSelector.test.js (hunk 1 above)
Comment (sunilb0575) [reply app/javascript/selectors/detailSelector.test.js]: @RamuKammagani-osi as both ```isAdminWithEditPrivileges``` and ```isUserLocked``` return values if any or else returns undefined and as those respective values are used only in ```isUserEditable``` method, I have tested null and undefined in ```isUserEditable``` method. 
And also UI is independent of the return values from those methods(isAdminWithEditPrivileges, isUserLocked).

File: app/javascript/selectors/detailSelector.test.js (hunk 2 above)
Comment (sunilb0575) [reply app/javascript/selectors/detailSelector.test.js]: https://github.com/ca-cwds/county-admin/pull/343#discussion_r262306614

File: app/javascript/selectors/detailSelector.test.js (hunk 1 above)
Comment (RamuKammagani-osi) [reply app/javascript/selectors/detailSelector.test.js]: Alright!, that sounds good.

**Pull request merged**